docs/book/src/crd/index.md

```diff
@@ -1145,6 +1145,18 @@ string
 </tr>
 <tr>
 <td>
+<code>stackTags</code><br/>
```

Comment:
Wondering why there are changes in the index.md.
Not seeing in my local when I build the book.

Reply:
I'm not 100% sure but it could be an issue with how Makefile targets work. Because the CRD files didn't change the index.md doesn't need updating. 
To double check I removed `docs/book/src/crd/index.md` and ran `make -C docs/book build` again. This produced the same output. 
If you try to do this, do you still have no changes or do you get the same output?

Reply:
If you prefer, I can also remove this change from this PR to ensure no unrelated change is added with this PR. LMK if I should do this.